web/src/submessage.js

```diff
@@ -1,8 +1,23 @@
+import z from "zod";
```

Comment:
Any reason for changing `import {z} from "zod";` in `zform.js` to `import z from "zod";`?

Reply:
The `zod/src/index.ts` makes a default export **as well as** a named export; so using either `import z ..` or `import {z} ..` should be fine.
```
import * as z from "./external";
export * from "./external";
export { z };
export default z;
```
Reference - https://github.com/colinhacks/zod/blob/master/src/index.ts

But I can gladly change that if you prefer so ?

Reply:
It's just that most files in the codebase import it as `{z}` so for consistency that would have been better, but I don't think it's important enough to warrant a fix. :)